### PR TITLE
Auth functionality/category filter

### PR DESCRIPTION
**Pull request by talrejah02**
Features Added
- login(form validation will be added later)
- signup(form validation will be added later)
- category filter
- private Route

### CODE REVIEW

**Review by Shrinkhlarajpoot — COMMENTED** [review]
Clean UI.. Code looks good.. Good to merge.

**Review by Shrinkhlarajpoot — APPROVED** [review]

File: src/components/navbar/nav.jsx [hunk 1]
```diff
@@ -1,78 +1,83 @@
...
-import watchlater from "../../assets/svg/watchlater.svg"
+import Home from "../../assets/svg/Home.svg";
+import Explore from "../../assets/svg/Explore.svg";
+import Like from "../../assets/svg/Like.svg";
+import history from "../../assets/svg/History.svg";
+import playlist from "../../assets/svg/playlist.svg";
+import Logo from "../../assets/svg/Logo.svg";
+import account from "../../assets/svg/account.svg";
+import Setting from "../../assets/svg/Setting.svg";
+import watchlater from "../../assets/svg/watchlater.svg";
```
Comment (pankaj-prog) [inline src/components/navbar/nav.jsx:11]: create index file in svg and import all svg's from it

File: src/components/navbar/nav.jsx [hunk 2]
```diff
@@ -1,78 +1,83 @@
...
 import "./nav.css";
+import { useAuth } from "../../context";
```
Comment (pankaj-prog) [inline src/components/navbar/nav.jsx:13]: use absolute imports

File: src/components/videoListing/videoListing.jsx [hunk 3]
```diff
@@ -2,14 +2,25 @@ import React from "react";
...
 export { Videolisting };
+// categoryName: "Backpacking"
+// created: "2016-02-26T04:52:35Z"
+// creator: "Indiahikes"
+// creatorImage: "Images/videoImages/creator.webp"
+// description: "There's just one rule you follow while packing your bag. The ABCD Rule. A - Accessibility, B - Balance, C - Compactness and D - Done the above. It's as simple as that."
+// id: "4"
+// image: "Images/videoImages/backpack.webp"
+// tags: ['trending']
+// title: "How to pack your backpack?"
```
Comment (pankaj-prog) [inline src/components/videoListing/videoListing.jsx:26]: remove these comment

**Review by pankaj-prog — APPROVED** [review]
Nice UI,
 nav links are not clickable when clicked around text, you could use padding for nav-btn instead of margin, then it will work fine

**Pull request merged**